capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/fileimport/FileChooserActivity.java

```diff
@@ -318,153 +281,27 @@ public void onItemSelected(@NonNull final ProvidersAppItem item) {
                         // Store the selected item, it is needed for requesting permission
                         // after showing the rationale
                         mSelectedAppItem = item;
```

Comment:
[Request] It looks like the `mSelectedAppItem` is not needed anymore. It was only needed to know which app to launch after the permission request. I think it is safe to remove it. Can you please double check and remove it, if possible? Thanks!